backend/src/emails/templates/addedContributionMessage/html.pug

```diff
@@ -0,0 +1,17 @@
+doctype html
+html(lang=locale)
+  head
+    title= t('emails.addedContributionMessage.subject')
+  body
+    h1(style='margin-bottom: 24px;')= t('emails.addedContributionMessage.subject')
+    #container.col
+      p(style='margin-bottom: 24px;')= t('emails.general.helloName', { firstName, lastName })
+      p= t('emails.addedContributionMessage.commonGoodContributionMessage', { senderFirstName, senderLastName, contributionMemo })
+      p= t('emails.addedContributionMessage.toSeeAndAnswerMessage')
```

Comment:
As well `message` was not used in the mail before, see: https://github.com/gradido/gradido/blob/e458a7dd8d061c4d56b8e448fefe8a0dd6c706d1/backend/src/mailer/sendAddedContributionMessageEmail.ts

I rebuild the mails like they where before, because I guess this was intentional.